charts/baremetal-csi-plugin/values.yaml

```diff
@@ -4,6 +4,7 @@ global:
 
 env:
  test: false
+ openshift: false
```

Comment:
please rename to `mountHostRoot` since this problem might occur on any platform.

Reply:
Fixed